docker/README.md

```diff
@@ -248,6 +248,42 @@ services:
 For more information, see the docs for
 [Docker Compose Extend](https://docs.docker.com/compose/extends/).
 
+### Snapshot Archival
```

Comment:
This should actually go at the top - the Features have all been added with the 'latest' at the top

Move this section up to line 82 and it looks good to me.